nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py

```diff
@@ -619,7 +649,7 @@ def __init__(
         Args:
             g2p: Grapheme to phoneme module, should be IpaG2p or some subclass thereof.
             locale: Locale used to determine default text processing logic and punctuation.
-                Supports ["en-US", "de-DE", "es-ES"]. Defaults to "en-US".
+                Supports ["en-US", "de-DE", "es-ES", "fr-FR"]. Defaults to "en-US".
```

Comment:
When adding new language support for `IPATokenizer`, I would suggest to go over its `__init__` function, especially Line 689, where `any_locale_text_preprocessing` is applied for all non en-US locales. `any_locale_text_preprocessing` will preserve the original text's cases. But it seems French needs all lowercased?